algorithms/indexing/test_lattice_search.py

```diff
@@ -99,3 +99,37 @@ def test_stills_indexer_BasisVectorSearch_i04_weak_data(
     assert indexed_experiments[0].crystal.get_unit_cell().parameters() == pytest.approx(
         (57.752, 57.776, 150.013, 90.0101, 89.976, 90.008), rel=1e-2
     )
+
+
+@pytest.mark.parametrize(
+    "indexing_method,space_group,unit_cell",
+    (("low_res_spot_match", "P422", (57.8, 57.8, 150.0, 90, 90, 90)),),
+)
+def test_stills_indexer_LatticeSearch_i04_weak_data(
+    i04_weak_data, indexing_method, space_group, unit_cell
+):
+    reflections = slice_reflections(i04_weak_data["reflections"], [(1, 2)])
+    experiments = slice_experiments(i04_weak_data["experiments"], [(1, 2)])
+    for experiment in experiments:
+        experiment.imageset = ImageSet(
+            experiment.imageset.data(), experiment.imageset.indices()
+        )
+        experiment.imageset.set_scan(None)
+        experiment.imageset.set_goniometer(None)
+        experiment.scan = None
+        experiment.goniometer = None
+    params = phil_scope.fetch().extract()
+    params.indexing.method = indexing_method
+    params.indexing.basis_vector_combinations.max_refine = 5
+    if unit_cell is not None:
```

Comment:
`unit_cell` and `space_group` are required input for this indexing method right? So in the check for if unit_cell/space_group is not None is not necessary.

Reply:
Yes, copy-pasta leftovers. An alternative would be to merge the `test_stills_indexer_LatticeSearch_i04_weak_data` and `test_stills_indexer_BasisVectorSearch_i04_weak_data` tests, by figuring out whether to import a `StillsIndexerBasisVectorSearch` or a `StillsIndexerLatticeSearch`

Reply:
I've merged the tests, which reduces code duplication. It also exercises the possibility that `BasisVectorSearch` gets passed an indexing method it is not aware of